README.md

```diff
@@ -256,10 +263,21 @@ If you like this project please __Star us on GitHub — it helps!__ :star: :star
 
 Please visit the [PcapPlusPlus web-site](https://pcapplusplus.github.io/community) to learn more.
 
+[![GitHub issues](https://img.shields.io/github/issues/seladb/PcapPlusPlus?style=flat&label=Issues&logo=github)](https://github.com/seladb/PcapPlusPlus/issues/new/choose)
+[![Google Groups](https://img.shields.io/badge/Google%20Groups-%234285F4.svg?&style=flat&logo=google&logoColor=white)](https://groups.google.com/d/forum/pcapplusplus-support)
+[![Stack Overflow](https://img.shields.io/badge/Stack%20Overflow-%23F58025.svg?&style=flat&logo=stackoverflow&logoColor=white)](https://stackoverflow.com/questions/tagged/pcapplusplus)
+[![E-Mail](https://img.shields.io/badge/E--Mail-%230078d4.svg?&style=flat&logo=minutemailer&logoColor=white)](mailto:pcapplusplus@gmail.com)
+[![Twitter Follow](https://img.shields.io/twitter/follow/seladb?label=Follow%20PcapPlusPlus&style=social)](https://twitter.com/intent/follow?screen_name=seladb)
+
 ## Contributing
 
 We would very much appreciate any contribution to this project. If you're interested in contributing please visit the [contribution page](https://pcapplusplus.github.io/community#contribute) in PcapPlusPlus web-site.
 
+![GitHub contributors](https://img.shields.io/github/contributors/seladb/PcapPlusPlus?style=flat&label=Contributors&logo=github)

```

Comment:
This is actually something we can add at the beginning of the README file, right next to CI status. What do you think?